### PR TITLE
Fix UTscapy logs sync

### PR DESCRIPTION
**Pull request by gpotter2**
- Use the same output stream in UTscapy.

I realized that we were using two separate stream outputs for UTscapy, which was the reason for the logs to be messed up.

### CODE REVIEW

**Comment by guedou** [comment]
I think that it will make more sense to output everything except errors to stdout.

**Comment by gpotter2** [comment]
I'm pretty sure that's historical, I don't really see a reason to keep it this way :/

File: scapy/tools/UTscapy.py [hunk 1]
```diff
@@ -590,21 +590,19 @@ def drop(scapy_ses):
         test_campaign.trunc(i + 1)
         test_campaign.interrupted = True
         if verb:
-            print("Campaign interrupted!", file=sys.stderr)
+            print("Campaign interrupted!")
```
Comment (guedou) [inline scapy/tools/UTscapy.py:593]: This should be kept as stderr.

File: scapy/tools/UTscapy.py [hunk 2]
```diff
@@ -824,7 +822,7 @@ def usage():
 -k <kw1>,<kw2>,...\t: include only tests with one of those keywords (can be used many times)
 -K <kw1>,<kw2>,...\t: remove tests with one of those keywords (can be used many times)
 -P <preexecute_python_code>
-""", file=sys.stderr)
```
Comment (guedou) [inline scapy/tools/UTscapy.py:827]: This should be kept as stderr.

File: scapy/tools/UTscapy.py [hunk 3]
```diff
@@ -838,8 +836,9 @@ def execute_campaign(TESTFILE, OUTPUTFILE, PREEXEC, NUM, KW_OK, KW_KO, DUMP, DOC
     try:
         test_campaign = parse_campaign_file(TESTFILE)
     except ValueError as ex:
-        print(theme.red("Error while parsing '%s': '%s'" % (TESTFILE.name, ex)),
-              file=sys.stderr)
+        print(
```
Comment (guedou) [inline scapy/tools/UTscapy.py:839]: This should be kept as stderr.

File: scapy/tools/UTscapy.py [hunk 4]
```diff
@@ -1032,7 +1031,7 @@ def main():
                 KW_KO.extend(optarg.split(","))
 
     except getopt.GetoptError as msg:
-        print("ERROR:", msg, file=sys.stderr)
+        print("ERROR:", msg)
```
Comment (guedou) [inline scapy/tools/UTscapy.py:1034]: This should be kept as stderr.

File: scapy/tools/UTscapy.py [hunk 5]
```diff
@@ -1077,11 +1076,11 @@ def main():
         collect_types.start()
 
     if VERB > 2:
-        print(" " + arrow + " Booting scapy...", file=sys.stderr)
+        print(" " + arrow + " Booting scapy...")
     try:
         from scapy import all as scapy
     except Exception as e:
-        print("[CRITICAL]: Cannot import Scapy: %s" % e, file=sys.stderr)
+        print("[CRITICAL]: Cannot import Scapy: %s" % e)
```
Comment (guedou) [inline scapy/tools/UTscapy.py:1083]: This should be kept as stderr.

File: scapy/tools/UTscapy.py [hunk 6]
```diff
@@ -1184,17 +1181,16 @@ def main():
     # Print end message
     if VERB > 2:
         if glob_result == 0:
-            print(theme.green("UTscapy ended successfully"), file=sys.stderr)
+            print(theme.green("UTscapy ended successfully"))
         else:
-            print(theme.red("UTscapy ended with error code %s" % glob_result),
-                  file=sys.stderr)
+            print(theme.red("UTscapy ended with error code %s" % glob_result))
```
Comment (guedou) [inline scapy/tools/UTscapy.py:1186]: This should be kept as stderr.

**Review by guedou — APPROVED** [review]
LGTM. I made new comments that are all similar: keep errors on stderr. You can decide to ignore them.

**Comment by gpotter2** [comment]
The issue if we use two different streams is that the logs might end up mixed up.
That's what's happening in some logs at the moment: the final "glob_output" summary gets injected while stderr isn't flushed yet.

**Pull request merged**